query-engine/core/src/executor/interpreting_executor.rs

```diff
@@ -1,17 +1,19 @@
 use super::{pipeline::QueryPipeline, QueryExecutor};
-use crate::{Operation, QueryGraphBuilder, QueryInterpreter, QuerySchemaRef, Response, Responses};
+use crate::{Operation, QueryGraphBuilder, QueryInterpreter, QuerySchemaRef, ResponseData};
 use async_trait::async_trait;
-use connector::{ConnectionLike, Connector};
+use connector::{Connection, ConnectionLike, Connector};
+use futures::future;
 
 /// Central query executor and main entry point into the query core.
 pub struct InterpretingExecutor<C> {
     connector: C,
     primary_connector: &'static str,
+
+    /// Flag that forces individual operations to run in a transaction.
+    /// Does _not_ force batches to use transactions.
```

Comment:
This should be documented, that if using pgBouncer, one must not run batches without setting the transaction flag on.

Reply:
Batches without `transaction: true` will behave as a collection of single queries, which _will_ honor the `force_transactions` flag. I don't think there's an issue here.

Reply:
Ah, I just misunderstood your comment. I thought this meant batches are not transactional at all in any level if not specified to be so.